README.md

```diff
@@ -231,7 +231,7 @@ type ChatPlugin struct{ *network.Plugin }
 func (state *ChatPlugin) Receive(ctx *network.PluginContext) error {
     switch msg := ctx.Message().(type) {
         case *messages.ChatMessage:
-            glog.Infof("<%s> %s", ctx.Client().ID.Address, msg.Message)
+            log.Info().Msgf("<%s> %s", ctx.Client().ID.Address, msg.Message)
```

Comment:
Is there going to be another PR to convert these logs into key value pairs? Or is this just a first draft

Reply:
done